### DISCUSSION TITLE
Column-level lineage capture

### DISCUSSION THREAD

**Discussion by lineclappe** [AbsaOSS/spline#863] [Q&A]
Hello,  I have a problem about spline. Can spline capture column-level lineage? I try the spline-ui and I can see the operations such as Project, Filter and SubqueryAlias. But I can not find the information about table column.

**Comment by wajda** [comment]
Table columns the same as the operation output attributes (or input attributes for read operations).

**Reply by wajda** [reply]
![image](https://user-images.githubusercontent.com/795479/114396649-11c20c80-9b9e-11eb-833b-87aeff710b16.png)

**Reply by lineclappe** [reply]
Thanks for your reply. 
I running spline-web-client with version 0.5.6.  The ui is not identical to your screenshot, should I upgrade the web-client and rest-server?  

And when I click on the certain operation, 500 status code was returned. I am not sure what the undelying problem is.

**Reply by wajda** [reply]
It's a screenshot from the new Spline UI that is replacing the old one starting from upcoming Spline 0.6.0.
Despite that I obviously would encourage you to try and switch to the new UI, both UIs are still using the same REST API, so both should be working fine (including the operation output schema info). 

> Just for the reference, in the version 0.5 the new UI is mostly a redesign, and in the version 0.6 it comes with even better and more flexile design and a few smaller features, while keeping the compatibility with the old one in majority of places.
> The old UI is on the end-of-life, and support for it will most likely be dropped in one of the next Spline versions when a new REST API is developed.

The new UI - https://github.com/AbsaOSS/spline-ui

If you see 500 it means something isn't right. Please file an issue and we'll look at it asap.
